libs/vectoria/src/embedding.service.ts

```
@@ -0,0 +1,133 @@
+import { pipeline } from '@huggingface/transformers';
+/**
+ * Service for generating embeddings using transformers.js
+ */
+export class EmbeddingService {
+  private model: any = null;
+  private pipeline: any = null;
+  private modelName: string;
+  private dimensions: number = 384; // default for all-MiniLM-L6-v2
```

ESLint

[error] Type number trivially inferred from a number literal, remove type annotation.

(@typescript-eslint/no-inferrable-types)